### PR TITLE
[Integration Test][omevv_firmware]: Updating single host in cluster

### PR DESCRIPTION
**Pull request by ABHISHEK-SINHA10**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| |

# ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### OUTPUT

```paste below

```
##### ADDITIONAL INFORMATION

```paste below

```
# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

**Comment by ansible-collections-svc** [comment]
Can one of the admins verify this patch?

**Comment by trisha-dell** [comment]
Please update the required details in the PR. 
Lint issues need to be fixed as well.

File: tests/integration/targets/omevv_firmware/files/ansible_doc.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+- '        This module allows you to create, modify, or delete an'
+- '        OpenManage Enterprise Integration for VMware Center (OMEVV)'
+- '        firmware repository profile.'
+- ''
+- 'ADDED IN: version 9.8.0 of dellemc.openmanage'
```
Comment (trisha-dell) [inline tests/integration/targets/omevv_firmware/files/ansible_doc.txt]: This should be 9.10.0

Comment (trisha-dell) [reply tests/integration/targets/omevv_firmware/files/ansible_doc.txt]: This document seems to be of omevv firmware repository profile, please update this.

Comment (ABHISHEK-SINHA10) [reply tests/integration/targets/omevv_firmware/files/ansible_doc.txt]: Document need to update, I have seen Reeti has given comment on dev PR, let it address and need build after this.

**Review by Kritika-Bhateja-03 — DISMISSED** [review]

File: tests/integration/targets/omevv_firmware/files/ansible_doc.txt (hunk 1 above)
Comment (ABHISHEK-SINHA10) [reply tests/integration/targets/omevv_firmware/files/ansible_doc.txt]: updated

**Review by trisha-dell — DISMISSED** [review]

**Review by Kritika-Bhateja-03 — DISMISSED** [review]

**Review by ShivamSh3 — DISMISSED** [review]

**Comment by ABHISHEK-SINHA10** [comment]
image

**Review by rajshekarp87 — APPROVED** [review]

**Review by Saksham-Nautiyal — APPROVED** [review]

**Review by Kritika-Bhateja-03 — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**